server/routes/getOverviewInfo.js

```diff
@@ -3,41 +3,41 @@ const config = require('../../config.js');
 
 const getProduct = (productID) => {
   if (!productID) { productID = 28212 }
-  return new Promise((resolve, reject) => {
-    axios({
-      method: 'GET',
-      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/products/${productID}`,
-      headers: {
-        'User-Agent': 'request',
-        'Authorization': config.TOKEN
-      }
-    })
-      .then(data => {
-        resolve(data.data)
-      })
-      .catch(err => {
-        reject(err)
-      })
+
+  return axios({
+    method: 'GET',
+    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/products/${productID}`,
+    headers: {
+      'User-Agent': 'request',
+      'Authorization': config.TOKEN
+    }
   })
+    .then(data => {
+      return data.data
```

Comment:
This is for me because I don't understand promises well... but what is the difference between resolve and return here?

Reply:
Resolve and Reject are arguments given names for a when we specify a Promise. 
Return and Resolve are by me in success cases. 
Return is the output of the promise. Resolve is what you use on success of the promise. 

I hope this is generally correct. I'm sure if a TM saw that they'd have a lot to say.